html/yogatherapy.html

```diff
@@ -75,34 +76,51 @@ <h1>Yoga Enhances Your Life</h1>
       </div>
     </header>
     <!-- Header Section Ends here -->
-
+  
     <!--Benefits section-->
     <section class="benefits" id="benefits">
-        <div class="col-md-12 text-center">
-            <h1><strong>Benefits of Yoga</strong></h1>
-        </div>
-        <div class="bfit">
-            <img src="https://preview.colorlib.com/theme/yogabest/images/xabout.png.pagespeed.ic.H7uxRdWOgu.webp"
-                width="700" height="600">
-            <div class="btext">
-                <div class="yoga-text"><i class="fa fa-dot-circle-o" aria-hidden="true"></i>
-                    <h3>Yoga benefits in <strong>weight loss</strong></h3></div>
-                    
-                    <div class="yoga-text"><i class="fa fa-dot-circle-o" aria-hidden="true"></i><h3>Yoga is one of the best solutions for <strong>stress relief</strong></h3></div>
-                    <div class="yoga-text"><i class="fa fa-dot-circle-o" aria-hidden="true"></i><h3>Yoga helps for <strong>inner peace</strong></h3></div>
-                    <div class="yoga-text"><i class="fa fa-dot-circle-o" aria-hidden="true"></i><h3>Yoga Improves <strong>Immunity</strong></h3></div>
-                    <div class="yoga-text"><i class="fa fa-dot-circle-o" aria-hidden="true"></i><h3>Practice of Yoga Offers <strong>Awareness</strong></h3></div>
-                    <div class="yoga-text"><i class="fa fa-dot-circle-o" aria-hidden="true"></i><h3>Yoga improves <strong>relationships</strong></h3></div>
-                    <div class="yoga-text"><i class="fa fa-dot-circle-o" aria-hidden="true"></i><h3>Yoga Increases <strong>Energy</strong></h3></div>
-                    <div class="yoga-text"><i class="fa fa-dot-circle-o" aria-hidden="true"></i><h3>Yoga Gives you Better <strong>Flexibility and Posture</strong></h3></div>
-                    <div class="yoga-text"><i class="fa fa-dot-circle-o" aria-hidden="true"></i><h3>Yoga helps in improving <strong>intuition</strong></h3></div>
-            </div>
-
+      <div class="col-md-12 text-center">
+        <h1><strong>Benefits of Yoga</strong></h1>
+      </div>
+      <div class="bfit">
+        <img src="https://preview.colorlib.com/theme/yogabest/images/xabout.png.pagespeed.ic.H7uxRdWOgu.webp" width="700"
```

Comment:
The code seems to be amazing but it's better to write the **width** and **height** of an image in CSS file.

``` diff
- <img src="" width="700" height="600">
+ img { width: 700px; height: 600px;}
```

> **Note**
> Why should we use it in CSS file?
1. For making the code to looks clear for code reader.
2. For making Responsive